app/views/service_types/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= params[:type] %> Businesses</h1>
+
+<% @businesses.each do |biz| %>
+	<div class='business'>
+		<%= link_to biz.name, "/#{params[:location]}/#{params[:service]}/#{params[:type]}/#{biz.id}" %>
```

Comment:
Could maybe replace with <%= link_to biz.name, business_show_path %> or something like that?